res/CircleRecipes/RoPE_000/test.recipe

```diff
@@ -0,0 +1,72 @@
+# This recipe is just to check processing of rope(rotary position embedding)
+# The rope op requires a sin/cos input, so add the sin/cos constant table was divided into 0 ~ 80 in -1 ~ 1
```

Comment:
How about'
```suggestion
# For testing RoPE op, `SIN` and `COS` values are given as tables with 80 items,
# input range from 0.0 to PAI resulting -1.0f to 1.0f
```

Reply:
> # input range from 0.0 to PAI resulting -1.0f to 1.0f
What does `PAI` mean?

Reply:
Ah.... 3.141492.... maybe use symbol $\pi$

Reply:
or use `PAI(3.141592...)` ?

Reply:
@seanshpark 
i updated it by referring to your comment,
thanks